graphql/resolvers.go

```diff
@@ -2755,16 +2761,9 @@ func (r *annotationResolver) WebhookConfigured(ctx context.Context, obj *restMod
 	return IsWebhookConfigured(t), nil
 }
 
-func (r *annotationResolver) CreatedIssues(ctx context.Context, obj *restModel.APITaskAnnotation) ([]*restModel.APIIssueLink, error) {
```

Comment:
these fields will resolve every time an Annotation gql type is queried as [part of APITaskAnnotationBuildFromService](https://github.com/evergreen-ci/evergreen/blob/main/rest/model/task_annotations.go#L125-L127) since these code changes remove the explicit resolvers from this file and gqlgen.yml.
do you think the linked code will result in expected/the same output as we had before when querying CreatedIssues, Issues or SuspectedIssues

Reply:
These fields are part of the annotation document, so if we query for any annotation field, I think it's fine to grab all the fields. Especially since the main task page needs the issues field.